components/CalendarNavigationBar/index.js

```diff
@@ -24,14 +24,14 @@ export default function CalendarNavigationBar({ page }) {
   return (
     <StyledHeader>
       <StyledLink
-        active={page === "today" ? true : false}
+        active={page === "today" ? "true" : "false"}
```

Comment:
```suggestion
        active={page === "today"}
```

Reply:
When giving a boolean value to the active prop i get an error message in the console saying: ```
Warning: Received `true` for a non-boolean attribute `active`.

If you want to write it to the DOM, pass a string instead: active="true" or active={value.toString()}.
a
```